### PR TITLE
Feat/log info

### PR DESCRIPTION
**Pull request by hwdtw**

### CODE REVIEW

File: src/main/java/com/rock_mc/syn/api/Log.java [hunk 1]
```diff
@@ -1,16 +1,75 @@
...
+            long[] times = Utils.parseTime(args);
+            List<String> players = Utils.parseUsers(args);
+
+            Timestamp start = null;
+            Timestamp end = null;
+            Integer page = null;
+            Integer rows = null;
+
+            LocalDateTime currentDateTime = LocalDateTime.now(ZoneId.of("UTC"));
+            Instant now = currentDateTime.atZone(ZoneId.of("Asia/Taipei")).toInstant();
```
Comment (PttCodingMan) [inline src/main/java/com/rock_mc/syn/api/Log.java:37]: Can we detect the timezone? Or set it in config?

File: src/main/java/com/rock_mc/syn/db/Database.java [hunk 2]
```diff
@@ -50,4 +53,10 @@ public abstract class Database {
     public abstract PluginPlayerInfo getPlayerByName(String playerName);
 
     public abstract String [] getBannedPlayerList();
+
+    public abstract void addLogEvent(String playerUUID, String eventName);
+
+    public abstract List<EventLog> getLogEvents(List<String> playerUUIDs, Timestamp start, Timestamp end, Integer page, Integer rows);
```
Comment (PttCodingMan) [inline src/main/java/com/rock_mc/syn/db/Database.java:59]: Why use Integer not int? what's the difference?

File: src/test/java/com/rock_mc/syn/CmdExecutorTest.java [hunk 3]
```diff
@@ -159,4 +159,20 @@ void tabComplete() {
...
+
+
+    @Test
+    void logCmd() {
+        PlayerJoinEvent.getHandlerList().unregister(plugin);
+
+        PlayerMock opPlayer = server.addPlayer();
+        opPlayer.setOp(true);
+
+        opPlayer.performCommand("syn log t:2h");
```
Comment (PttCodingMan) [inline src/test/java/com/rock_mc/syn/CmdExecutorTest.java:171]: Can we add more test case about time parsing?
For example: t:1d, t:5m

File: src/main/java/com/rock_mc/syn/api/Log.java (hunk 1 above)
Comment (hwdtw) [reply src/main/java/com/rock_mc/syn/api/Log.java:37]: To better handle when the server's default time zone is not UTC, replaced the code by
`Instant now = currentDateTime.atZone(TimeZone.getDefault().toZoneId()).toInstant();`

**Review by PttCodingMan — APPROVED** [review]

**Pull request merged**